samples/msal-node-samples/Managed-Identity/ServiceFabric/index.ts

```diff
@@ -0,0 +1,98 @@
+import { LoggerOptions } from "@azure/msal-common";
```

Comment:
Is there anything different in this sample vs the IMDS sample? (except the Readme?). Should these be consolidated?

Reply:
The use of expressjs (and the README) are the only differences. Express is needed to serve the application to the Service Fabric's website.

Reply:
The IMDS sample is a standalone console app vs this one is an app/ website that can be deployed to an app service or service fabric